Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM ubuntu:20.04
+ARG DATAVERSE_API_TOKEN
+
+# Label
+LABEL org.opencontainers.image.title="metadata generation for fish image drexel version"
+LABEL org.opencontainers.image.authors=" J. Pepper, K. Karmani, T. Tabarin"
+LABEL org.opencontainers.image.source="https://github.com/hdr-bgnn/drexel_metadata"
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install some basic utilities
+RUN apt-get update && apt-get install -y \
+    curl \
+    ca-certificates \
+    sudo \
+    git \
+    bzip2 \
+    libx11-6 \
+    wget \
+    build-essential \
+    libglib2.0-0 \
+    libsm6 \
+    libxext6 \
+    libxrender-dev \
+    sudo \
+    cmake \
+    ninja-build \
+ && rm -rf /var/lib/apt/lists/*
+
+# Create a working directory
+RUN mkdir /pipeline
+WORKDIR /pipeline
+
+# Create a non-root user and switch to it
+RUN adduser --disabled-password --gecos '' --shell /bin/bash user \
+ && chown -R user:user /pipeline
+RUN echo "user ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/90-user
+USER user
+
+# All users can use /home/user as their home directory
+ENV HOME=/home/user
+RUN chmod 777 /home/user
+
+# Set up the Conda environment
+ENV CONDA_AUTO_UPDATE_CONDA=false \
+    PATH=/home/user/miniconda/bin:$PATH
+RUN curl -sLo ~/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-py39_4.10.3-Linux-x86_64.sh \
+ && chmod +x ~/miniconda.sh \
+ && ~/miniconda.sh -b -p ~/miniconda \
+ && rm ~/miniconda.sh \
+ && conda clean -ya
+
+RUN pip install numpy pandas pynrrd pillow scikit-image jedi==0.17.2 opencv-python-headless pyDataverse==0.3.1
+
+# Detectron2 prerequisites
+RUN pip install torch==1.8.1+cpu torchvision==0.9.1+cpu -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
Why is this Dockerfile using older versions of pytorch/torchvision than the Pipefile already existing in the repo?
https://github.com/hdr-bgnn/drexel_metadata/blob/a8720c3bfefcbd5caf7d948fad4ba90c2adc7f3c/Pipfile#L27-L28

Reply:
In short, the version I used in the Dockerfile is what has worked for me.... I spend quiet sometime testing different version and that what worked. In particular during the development I didn't have access to a GPU and I had to use cpu. Additionally the use of GPU is a bit restrictive (at least it was to me) and since we are using this code as prediction the GPU is not as fundamental.
Bottom line : Everything I did in the changes for this branch is to make my workflow BGNN works.